xml/x509_issuers.xml

```diff
@@ -0,0 +1,131 @@
+    <param pos="0" name="os.product" value="Android"/>
+    <param pos="0" name="hw.device" value="Tablet"/>
+    <param pos="0" name="hw.vendor" value="NVIDIA"/>
+    <param pos="0" name="hw.product" value="SHIELD"/>
+  </fingerprint>  
+  <fingerprint pattern="^CN=NVidia Darcy NVidia Tegra K1-Denver Cast ICA,OU=Widevine,O=Google Inc,L=Kirkland,ST=Washington,C=US$">
+    <description>NVIDIA SHIELD (Android) with Google Cast (Darcy)</description>
+    <example>CN=NVidia Darcy NVidia Tegra K1-Denver Cast ICA,OU=Widevine,O=Google Inc,L=Kirkland,ST=Washington,C=US</example>
+    <param pos="0" name="os.vendor" value="Google"/>
+    <param pos="0" name="os.family" value="Linux"/>
+    <param pos="0" name="os.product" value="Android"/>
+    <param pos="0" name="hw.device" value="Tablet"/>
+    <param pos="0" name="hw.vendor" value="NVIDIA"/>
+    <param pos="0" name="hw.product" value="SHIELD"/>
+  </fingerprint>
+  <!-- End of Chromecast -->
+  <fingerprint pattern="^CN=Yealink Equipment Issuing CA,OU=yealink\.com,O=Yealink Network Technology Co\.\\,Ltd.,L=Xiamen,ST=Fujian,C=CN$">
+    <description>Yealink VoIP Phone</description>
+    <example>CN=Yealink Equipment Issuing CA,OU=yealink.com,O=Yealink Network Technology Co.\,Ltd.,L=Xiamen,ST=Fujian,C=CN</example>
+    <param pos="0" name="os.vendor" value="Yealink"/>
+    <param pos="0" name="os.family" value="Linux"/>
+    <param pos="0" name="os.product" value="Linux"/>
+    <param pos="0" name="hw.device" value="IP Phone"/>
+    <param pos="0" name="hw.vendor" value="Yealink"/>
+  </fingerprint>
+  <fingerprint pattern="^CN=[a-zA-Z0-9]+,OU=Internally Generated Certificate,O=American Power Conversion Corp,L=Default Locality,ST=Default State,C=US$">
+    <description>APC UPS</description>
+    <example>CN=ZA1117619249,OU=Internally Generated Certificate,O=American Power Conversion Corp,L=Default Locality,ST=Default State,C=US</example>
+    <param pos="0" name="hw.device" value="Power Device"/>
+    <param pos="0" name="hw.vendor" value="APC"/>
+  </fingerprint>
+  <fingerprint pattern="^CN=Temporary CA [a-fA-F0-9]{8}\-[a-fA-F0-9]{4}\-[a-fA-F0-9]{4}\-[a-fA-F0-9]{4}\-[a-fA-F0-9]{12},OU=Temporary CA">
+    <description>Cisco Video Communication Server</description>
+    <example>CN=Temporary CA 218131fe-8af4-11e7-aa6e-9950d6bbaf74,OU=Temporary CA 218131fe-8af4-11e7-aa6e-9950d6bbaf74,O=Temporary CA 218131fe-8af4-11e7-aa6e-9950d6bbaf74</example>
+    <param pos="0" name="hw.device" value="Video Conference"/>
+    <param pos="0" name="hw.vendor" value="Cisco"/>
+    <param pos="0" name="hw.product" value="TelePresence"/>
+  </fingerprint>
+  <fingerprint pattern="^O=VMware Installer$">
+    <description>VMWare ESX</description>
+    <example>O=VMware Installer</example>
+    <param pos="0" name="os.vendor" value="VMWare"/>
+    <param pos="0" name="os.product" value="ESX"/>
+    <param pos="0" name="os.device" value="Hypervisor"/>
+    <param pos="0" name="os.cpe23" value="cpe:/o:vmware:esx:-"/>
```

Comment:
Any indication that this is ESX `cpe:/o:vmware:esx:-` and not ESXi `cpe:/o:vmware:esxi:-`?

As a note, once this lands I'll try to add the ESXi host from my lab.

Reply:
These were indeed ESXi not ESX, updating.